portal/src/graphql/portal/PasswordlessAuthenticatorScreen.tsx

```diff
@@ -153,8 +158,18 @@ const PasswordlessAuthenticator: React.FC<PasswordlessAuthenticatorProps> = func
         state.smsTemplate !== initialState.smsTemplate
           ? state.smsTemplate
           : undefined,
-    }).catch(() => {});
-  }, [state, rawAppConfig, initialState, updateAppAndEmailSmsTemplatesConfig]);
+    })
+      .then(() => {
+        refetchAppAndEmailSmsTemplatesConfig().catch(() => {});
```

Comment:
I believe Apollo should be able to update the data state after mutation (updateAppAndEmailSmsTemplatesConfig). Is it not working?

Reply:
@axolotl0212 The email and sms templates are obtained by `rawConfigFile(path)`, so they are not updated after mutation. Maybe returning templates in `UpdateAppAndEmailSmsTemplatesConfigMutation` mutation can trigger Apollo to update those templates?

Reply:
@axolotl0212 Fixed. Included templates in mutation return object so that Apollo can update the templates.